package.json

```diff
@@ -37,11 +37,12 @@
     "winston": "^3.3.3"
   },
   "scripts": {
-    "start": "PORT=3001 react-app-rewired start",
+    "start": "yarn run downloadCustomTypes && PORT=3001 react-app-rewired start",
```

Comment:
```suggestion
    "start": "yarn downloadCustomTypes && PORT=3001 react-app-rewired start",
```

Reply:
I wonder if we could make the script be run in `pre-` hooks, but it would have to be `pre-start`, `pre-lint` and `pre-build`, so maybe easier to be explicit on the call site.

Can you update the readme as well and mention that it's required to fetch the files? Cause the error messages might be quite confusing (i.e. missing file).

Reply:
i can add the prestart , which is supported in create-react-app , but for the build just having the step configured in the github action i think it does the job. Also i like be explicit about this dependency in the logs as a separate step.